extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -56,7 +56,7 @@ export const buildOpenDefaultAttachmentElement = (element) =>
 export const buildOpenInlineAttachmentElement = (element) =>
   htmlFor(element)`
     <a class="
-        i-amphtml-story-page-open-attachment i-amphtml-story-system-reset"
+        i-amphtml-story-page-open-attachment i-amphtml-story-system-reset amp-story-page-attachment-ui-v2"
```

Comment:
we still need to add the prefix i-amphtml to ensure publishers can't target it

Reply:
Added it in all other areas that class is used too.